brainstormforce/ultimate-addons-for-gutenberg
# fix: Dynamic CSS/JS generation.

### Description
Improved CSS/JS file generation.

### Types of changes
Bugfix (non-breaking change which fixes an issue)

### How has this been tested?
- It is backward compatible.
- Delete files from the uag-plugin folder and refresh the pages, the files should be generated again.
- Create a new Page and Files should be generated for it too.
- Check by changing permissions to 555, still, the files should be generated.
- Check by changing permissions to 444, the fallback should work and inline styles should be generated instead of file generation.
- Also, No duplicated files should be generated if files are already present.
- Also, check with WooCommerce Pages and Archive Pages.

### Checklist:
